docs/src/content/docs/hi/guides/css-and-tailwind.mdx

```diff
@@ -0,0 +1,289 @@
+    <TabItem label="pnpm">
+
+    ```sh
+    pnpm install @astrojs/starlight-tailwind
+    ```
+
+    </TabItem>
+
+    <TabItem label="Yarn">
+
+    ```sh
+    yarn add @astrojs/starlight-tailwind
+    ```
+
+    </TabItem>
+
+    </Tabs>
+
+3.  Tailwind की मूल शैलियों के लिए एक CSS फ़ाइल बनाएं, उदाहरण के लिए `src/tailwind.css` पर:
+
+    ```css
+    /* src/tailwind.css */
+    @tailwind base;
+    @tailwind components;
+    @tailwind utilities;
+    ```
+
+4.  अपनी Tailwind आधार शैलियों का उपयोग करने और डिफ़ॉल्ट आधार शैलियों को अक्षम करने के लिए अपनी Astro कॉन्फ़िगरेशन फ़ाइल को अपडेट करें:
+
+    ```js {11-12,16-17}
+    // astro.config.mjs
+    import { defineConfig } from 'astro/config';
+    import starlight from '@astrojs/starlight';
+    import tailwind from '@astrojs/tailwind';
+
+    export default defineConfig({
+    	integrations: [
+    		starlight({
+    			title: 'Docs with Tailwind',
+    			customCss: [
+    				// Path to your Tailwind base styles:
+    				'./src/tailwind.css',
+    			],
+    		}),
+    		tailwind({
+    			// Disable the default base styles:
+    			applyBaseStyles: false,
+    		}),
+    	],
+    });
+    ```
+
+5.  Starlight Tailwind प्लगइन को `tailwind.config.mjs` में जोड़ें:
+
+    ```js ins={2,7}
+    // tailwind.config.mjs
+    import starlightPlugin from '@astrojs/starlight-tailwind';
+
+    /** @type {import('tailwindcss').Config} */
+    export default {
+    	content: ['./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}'],
+    	plugins: [starlightPlugin()],
+    };
+    ```
+
+### Tailwind के साथ Starlight को स्टाइल करें
+
+Starlight अपने UI में आपके [Tailwind थीम कॉन्फिगरेशन](https://tailwindcss.com/docs/theme) से मूल्यों का उपयोग करेगा।
+
+यदि सेट किया गया है, तो निम्नलिखित विकल्प Starlight की डिफ़ॉल्ट शैलियों क अवहेलन कर देंगे:
+
+- `colors.accent` — लिंक और वर्तमान वस्तु हाइलाइटिंग के लिए उपयोग किया जाता है
+- `colors.gray` — पृष्ठभूमि रंगों और सीमाओं के लिए उपयोग किया जाता है
+- `fontFamily.sans` — UI और सामग्री पाठ के लिए उपयोग किया जाता है
+- `fontFamily.mono` — कोड उदाहरणों के लिए उपयोग किया जाता है
+
+```js {12,14,18,20}
+// tailwind.config.mjs
+import starlightPlugin from '@astrojs/starlight-tailwind';
+import colors from 'tailwindcss/colors';
+
+/** @type {import('tailwindcss').Config} */
+export default {
+	content: ['./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}'],
+	theme: {
+		extend: {
+			colors: {
+				// आपका पसंदीदा उच्चारण रंग. Indigo Starlight के डिफॉल्ट्स के सबसे करीब है।
+				accent: colors.indigo,
+				// आपका पसंदीदा ग्रे स्केल। Zinc Starlight के डिफॉल्ट्स के सबसे करीब है।
+				gray: colors.zinc,
+			},
+			fontFamily: {
+				// आपका पसंदीदा टेक्स्ट फॉण्ट। Starlight डिफ़ॉल्ट रूप से system फ़ॉन्ट स्टैक का उपयोग करता है।
+				sans: ['"Atkinson Hyperlegible"'],
+				// आपका पसंदीदा कोड फॉण्ट। Starlight डिफ़ॉल्ट रूप से system monospace फॉण्ट का उपयोग करता है।
+				mono: ['"IBM Plex Mono"'],
+			},
+		},
+	},
+	plugins: [starlightPlugin()],
+};
+```
+
+## Theming
```

Comment:
Just to double check, is this left in English on purpose and it makes more sense instead of translating this heading?

Reply:
Human error, changing it